content/maxwell3_fdem/sphere_freespace/index.rst

```diff
@@ -0,0 +1,58 @@
+.. _sphere_freespace_index
+
+Response from a Conductive and Magnetically Permeable Sphere
+------------------------------------------------------------
+
+**Purpose**: Here, we explore the nature of frequency-domain responses generated by compact ore-bodies and unexploded ordnance items.
+If contained within a resistive media, this may be accomplished by considering the dipole response from a conductive and magnetically permeable sphere in free-space.
+The associated analytic expressions are provided by Wait (1951).
```

Comment:
Nice, this is a clear purpose statement. 

For the references, could you use ``:cite:`citekey` `` and add the citation to the reference.rst ?